content/en/docs/gke/deploy/deploy-cli.md

```diff
@@ -180,6 +180,13 @@ To deploy kubeflow, run the following command:
 
      * This issue is being tracked in [kubeflow/manifests#1234](https://github.com/kubeflow/manifests/issues/1234)
 
+   * If resources can't be created with error message like:
+    ```
+    error: unable to recognize ".build/application/app.k8s.io_v1beta1_application_application-controller-kubeflow.yaml": no matches for kind "Application" in version "app.k8s.io/v1beta1”
+    ```
+
+    This is expected race condition and you can try rerun `make apply` again, because Kubernetes API server hasn't been ready for CRDs when their instances were applied. This can happen multiple times for different resource kinds.
```

Comment:
Suggested revision:

This is expected race condition and you can try rerun `make apply` again, because Kubernetes API server hasn't been ready for CRDs when their instances were applied. This can happen multiple times for different resource kinds.

->

This issue occurs when the Kubernetes API server wasn't ready when a CRD was applied. This issue can happen multiple times for different kinds of resource. To resolve this issue, try running `make apply` again.

Reply:
I adjusted wording because there are some concept mismatch.
ref: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#create-a-customresourcedefinition